frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/embedding/common.embedding.unit.spec.tsx

```diff
@@ -471,7 +473,127 @@ describe("[OSS] embedding settings", () => {
     });
 
     expect(screen.getByText("Static embedding")).toBeInTheDocument();
-    expect(screen.getByText("Embedding SDK for React")).toBeInTheDocument();
+    expect(screen.getByText("Embedded analytics SDK")).toBeInTheDocument();
     expect(screen.getByText("Interactive embedding")).toBeInTheDocument();
   });
+
+  describe("self-hosted (OSS)", () => {
```

Comment:
Most of the new tests are based on plan, so I created a new describe block with different setup instead.

- common.* -> oss
- enterprise.* -> starter
- premium.* -> pro-self-hosted, pro-cloud

Try to diff these tests in your diff tool, or editor of your choice, because they're structured the same.